T2D-321 GCF accessions from Entrez API (#62)
EBIvariation/metadata-ws: docs/reference-sequence-import.md

The reference sequence XML is retrieved via ENA API like other objects
 
### GENOME_ASSEMBLY XML

> **For accessions starting with "GCF_"** the genome_assembly XML is retrieved with the following 2 endpoints from the 

Reviewer: ```suggestion
**For accessions starting with "GCF_"** (issued by NCBI RefSeq) the genome_assembly XML is retrieved with the following 2 endpoints from the 
```

> NBCI's Entrez API, We can search by the GCF accession, then retrieve its associated metadata.                        

Reviewer: ```suggestion
NBCI's Entrez API. The first one searches by the GCF accession, and the second one retrieves its associated metadata.                        
```

> To obtain the internal ID based on the GCF:https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=assembly&term=GCF_000442705.1                                                                 

Reviewer: ```suggestion
To obtain the internal ID based on the GCF: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=assembly&term=GCF_000442705.1                                                                 
```

> To query based on said internal ID:https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=assembly&id=49321

Reviewer: ```suggestion
To query based on said internal ID: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=assembly&id=49321
```

The reference sequence XML is retrieved via ENA API like other objects
 
### GENOME_ASSEMBLY XML
**For accessions starting with "GCF_"** (issued by NCBI RefSeq) the genome_assembly XML is retrieved with the following 2 endpoints from the 
NBCI's Entrez API. The first one searches by the GCF accession, and the second one retrieves its associated metadata.                        
                                    
To obtain the internal ID based on the GCF: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=assembly&term=GCF_000442705.1                                                                 

To query based on said internal ID: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=assembly&id=49321

The above endpoints are rate limited by max 3 requests per second, to use the endpoints seamlessly we have obtained an 

> apiKey as suggested here https://ncbiinsights.ncbi.nlm.nih.gov/2017/11/02/new-api-keys-for-the-e-utilities/

Reviewer: ```suggestion
API key as suggested here https://ncbiinsights.ncbi.nlm.nih.gov/2017/11/02/new-api-keys-for-the-e-utilities/
```
